ehitchhiking-ui/src/app/shared/components/menu/menu.component.ts

```diff
@@ -62,4 +65,44 @@ export class MenuComponent implements OnInit {
 	openProfileDialog(): void {
 		this.dialog.open(ProfileModalComponent, {panelClass: 'mat-dialog-no-padding', autoFocus: false});
 	}
+
+	private returnClass(type: NotificationTypes): string {
+		return NotificationComponent.headerClassMap[type];
+	}
+
+	callErrorNotification(notificationMessage: string) {
+		this.notificationBar.openFromComponent(NotificationComponent, {
+			duration: MenuComponent.notificationDuration,
+			data: {
+				message: notificationMessage,
+				type: NotificationTypes.error,
+			},
+			panelClass: [this.returnClass(NotificationTypes.error), 'notification-container'],
```

Comment:
pls create a constant for 'notification-container'

Reply:
the same for horizontalPosition: 'right'

Reply:
done